app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -133,4 +155,13 @@ def build_connect_params(options)
 
     connect_params
   end
+
+  def host_handle(vm, options)
```

Comment:
@jerryk55 - if `vm.host` is the HyperV host of the VM, then that should be the `Host` model object for the host. I'm assuming the relationship is similar to that of VMware - where SCVMM is like vSphere and HyperV is like ESX. So, if the credentials for HyperV hosts can be different than those of their associated SCVMM, we need to maintain credentials for each HyperV host - like we do for ESX. Then each host `has_many` authentications, just as the SCVMM EMS does.